samples/excel/16-custom-functions/basic-function.yaml

```diff
@@ -0,0 +1,22 @@
+order: 1
+id: excel-custom-functions-basic
+name: Basic custom function
+description: A simple custom function that calculates the volume of a sphere.
+host: EXCEL
+api_set:
+    CustomFunctionsRuntime: 1.1
+script:
+    content: |
+        /**
+         * Calculates the volume of a sphere. 
+         * @customfunction
+         * @param {number} radius
+         */
```

Comment:
Do you also want to show the result in the JS doc comments?

Reply:
Fixed!